lib/vtls/schannel_int.h

```diff
@@ -53,6 +53,16 @@
 #define CERT_ALT_NAME_IP_ADDRESS 8
 #endif
 
+#if defined(_MSC_VER) && (_MSC_VER <= 1600)
```

Comment:
Isn't that related to the Windows SDK version rather than the compiler version?

Reply:
It's a recurring warning, so I took the previous fix to this case:
8cfc936f5c9d1fe9a99956a916f95e137476b27c #10674 #6284

It also happened in libssh2, and got fixed with the same fix (albeit with <1700 instead of <=1600)
https://github.com/libssh2/libssh2/commit/2e3e0be816b2e71f59a6f56a0d38898ceeb5c82f
https://github.com/libssh2/libssh2/pull/925

Hard to say, but it could be a compiler regression causing this bogus warnings. It may also be a combination of compiler version and SDK. It's paired with 7.1 in this particular case.